README.md

```diff
@@ -83,12 +70,13 @@ output will always be written to disk, defaulting to `results.csv`.
   --no-smtp-cache             Do not cache SMTP results during the run.  This
                               may results in slower scans due to testing the
                               same mail servers multiple times.
-  --mx                        Only check mx records
-  --starttls                  Only check mx records and STARTTLS support.
+  --mx                        Only check MX records
+  --starttls                  Only check MX records and STARTTLS support.
                               (Implies --mx.)
-  --spf                       Only check spf records
-  --dmarc                     Only check dmarc records
-  --debug                     Output should include error messages.
+  --spf                       Only check SPF records
+  --dmarc                     Only check DMARC records
+  --json                      Output is in json format (default csv)
```

Comment:
In for a penny, in for a pound:
```suggestion
  --json                      Output is in JSON format (default CSV)
```

Reply:
This was actually copied from https://github.com/cisagov/trustymail/blob/3d45f40e9686563bb374570b281a520abbb9ee00/src/trustymail/cli.py#L9-L45. I'll make the change in both so it's still consistent.

Reply:
Please see [e6d1098](https://github.com/cisagov/trustymail/pull/127/commits/e6d1098099784d6a57d112d1df51e80c43d989f0).